.evergreen/install-dependencies.sh

```diff
@@ -1,105 +1,104 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -o errexit  # Exit the script with error if any of the commands fail
 
-NVM_WINDOWS_URL="https://github.com/coreybutler/nvm-windows/releases/download/1.1.9/nvm-noinstall.zip"
-NVM_URL="https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh"
-
 NODE_LTS_NAME=${NODE_LTS_NAME:-fermium}
-NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY}/node-artifacts"
-
-# this needs to be explicitly exported for the nvm install below
-export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
-export XDG_CONFIG_HOME=${NODE_ARTIFACTS_PATH}
-
-# create node artifacts path if needed
-mkdir -p "${NODE_ARTIFACTS_PATH}"
-
-function node_lts_to_version() {
-  case $1 in
-    "fermium")
-      echo 14
-      ;;
-    "gallium")
-      echo 16
-      ;;
-    "hydrogen")
-      echo 18
-      ;;
-    "iron")
-      echo 20
-      ;;
-    "latest")
-      echo 'latest'
-      ;;
-    *)
-      echo "Unsupported Node LTS version $1"
-      ;;
-  esac
-}
-
-function latest_version_for_node_major() {
-  local __NODE_MAJOR_VERSION=$1
-  local NODE_DOWNLOAD_URI="https://nodejs.org/download/release/latest-v${__NODE_MAJOR_VERSION}.x/SHASUMS256.txt"
-
-  if [ $__NODE_MAJOR_VERSION == 'latest' ]
-  then
-    NODE_DOWNLOAD_URI="https://nodejs.org/download/release/latest/SHASUMS256.txt"
-  fi
-
-  # check that the requested version does exist
-  curl --silent --fail $NODE_DOWNLOAD_URI &> /dev/null
-
-  echo $(curl --retry 8 --retry-delay 5  --max-time 50 --silent -o- $NODE_DOWNLOAD_URI | head -n 1 | awk '{print $2};' | cut -d- -f2)
-}
-
-NODE_MAJOR_VERSION=$(node_lts_to_version $NODE_LTS_NAME)
-NODE_VERSION=$(latest_version_for_node_major $NODE_MAJOR_VERSION)
-NODE_VERSION=${NODE_VERSION:1} # :1 gets rid of the leading 'v'
-
-echo "set version to $NODE_VERSION"
-
-# output node version to expansions file for use in subsequent scripts
+NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY:-$(pwd)}/node-artifacts"
```

Comment:
Would it be better to make this required? If project directory is not specified, something is wrong. Defaulting to the current working directory silently doesn't feel like the best approach

Reply:
PROJECT_DIRECTORY is calculated from pwd. I don't think we need to make it a prereq, this makes it easier to run the script locally